python/feathub/processors/flink/job_submitter/feathub_job_descriptor.py

```diff
@@ -69,6 +70,57 @@ def __init__(
         self.local_registry_tables = local_registry_tables
         self.allow_overwrite = allow_overwrite
         self.props = props
+        self.to_json = append_metadata_to_json(  # type: ignore
+            self.to_json,
+            self.__class__,
+        )
+
+    def to_json(self) -> Dict:
```

Comment:
Do we need to add these two method for `FeathubJobDescriptor`? I suppose we don't need to store FeathubJobDescriptor in registry.